reactDashboard/src/components/RouteMapRules/RouteMapRules.css

```diff
@@ -14,10 +14,6 @@
     height: auto;
     padding: 3%;
   }
-
-  h2 {
-    font-size: 1.2em;
```

Comment:
Could not find what this was doing being targeted inside @media. Moved it outside of @media in order to target the banner text. Used the same CSS file for the new component

Reply:
![image](https://user-images.githubusercontent.com/22414785/96510162-05936d00-1255-11eb-8543-137feb0b6b82.png)

Reply:
The media queries are styling the breakpoints for smaller devices - the overview banner should continue to display like the image above on smaller devices